app/code/Magento/MysqlMq/README.md

```diff
@@ -1,3 +1,24 @@
-# MysqlMq
+# Magento_MysqlMq module
 
-**MysqlMq** provides message queue implementation based on MySQL.
+**Magento_MysqlMq** provides message queue implementation based on MySQL.
+
+## Installation
```

Comment:
Please also add information about the recurring script populating the queue table to this section